aip/general/0193.md

```diff
@@ -138,6 +138,17 @@ descriptions, and examples:
     `ErrorInfo`,`LocalizedMessage`, and so on.
   - *Example*: Described in the [ErrorDetails](#error-details) section.
 
+
+**Important:** In the context of the of the `google.rpc.Status` Protobuf
+message, the value of the field `code` is the numeric equivalent of the
+enum value chosen from [`google.rpc.Code`][Code]. When the error is
+expressed as JSON, as in the [sample above](#json-representation), the
+value of the field by the same name, "code", is the HTTP status code
```

Comment:
Nit: Is it worth code formatting the JSON key for code? Since the protobuf field is code formatted...here and below
```suggestion
value of the field by the same name, `"code"`, is the HTTP status code
```

Reply:
Done.